src/e2e-test/features/BQExecuter/BQTableCreate.feature

```diff
@@ -0,0 +1,61 @@
+@BQExecute
+Feature: Executes a SQL query in BigQuery Execute
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST
+  Scenario: Verify BigQuery Execute Query
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Enter the BigQuery Execute query "bqexecuteQueryUC1"
+    Then Enter BigQuery Execute property datasetProjectId "projectId"
+    Then Enter BigQuery Execute property dataset "dataset"
+    Then Enter BigQuery Execute source property table name
+    Then Validate "BigQuery Execute" plugin properties
+    Then Close the BigQuery Execute Properties
+    Then Save the pipeline
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
```

Comment:
Add validation step in all scenarios
Validate records transferred to target BigQuery Table is equal to number of records fetched from SQL query "bqExecuteQuery"